jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/config/FastJson1IT.java

```diff
@@ -0,0 +1,115 @@
+import static org.junit.Assert.assertThat;
+
+public class FastJson1IT {
+    @Rule
+    public Jsonschema2PojoRule schemaRule = new Jsonschema2PojoRule();
+
+    @Test
+    @SuppressWarnings({"rawtypes", "unchecked"})
+    public void annotationStyleFastJsonProducesFastJsonAnnotations() throws ClassNotFoundException, SecurityException, NoSuchMethodException {
+
+        Class generatedType = schemaRule.generateAndCompile("/json/examples/torrent.json", "com.example",
+                        config("annotationStyle", "fastjson",
+                                "propertyWordDelimiters", "_",
+                                "sourceType", "json"))
+                .loadClass("com.example.Torrent");
+
+        assertThat(schemaRule.getGenerateDir(), not(containsText("jakarta.json.bind.annotation")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("javax.json.bind.annotation")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("org.codehaus.jackson")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("com.fasterxml.jackson")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("com.google.gson")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("@SerializedName")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("com.squareup.moshi")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("@com.squareup.moshi.Json")));
+        assertThat(schemaRule.getGenerateDir(), containsText("com.alibaba.fastjson"));
+        assertThat(schemaRule.getGenerateDir(), containsText("@JSONField"));
+
+        Method getter = generatedType.getMethod("getBuild");
+
+        assertThat(generatedType.getAnnotation(JsonPropertyOrder.class), is(nullValue()));
+        assertThat(generatedType.getAnnotation(JsonInclude.class), is(nullValue()));
+        assertThat(getter.getAnnotation(JsonProperty.class), is(nullValue()));
+    }
+
+    @Test
+    public void annotationStyleFastJsonMakesTypesThatWorkWithFastJson() throws ClassNotFoundException, SecurityException, IOException {
+
+        ClassLoader resultsClassLoader = schemaRule.generateAndCompile("/json/examples/", "com.example",
+                config("annotationStyle", "fastjson",
+                        "propertyWordDelimiters", "_",
+                        "sourceType", "json",
+                        "useLongIntegers", true));
+
+        assertJsonRoundTrip(resultsClassLoader, "com.example.Torrent", "/json/examples/torrent.json");
+        assertJsonRoundTrip(resultsClassLoader, "com.example.GetUserData", "/json/examples/GetUserData.json");
+    }
+
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    @Test
+    public void enumValuesAreSerializedCorrectly() throws ClassNotFoundException, NoSuchMethodException, SecurityException, InstantiationException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
- `"rawtypes"` could be removed if `Class` would be replaced with `Class<?>`
- `"unchecked"` could be removed if `JSON.parseObject(json, Map.class)` would be replaced with `JSON.<Map<String, String>>parseObject(json, Map.class)`
- `ClassNotFoundException, NoSuchMethodException, SecurityException, InstantiationException, IllegalAccessException, IllegalArgumentException, InvocationTargetException` could be replaced with `ReflectiveOperationException`